packages/ui/src/components/label.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+
+import * as LabelPrimitive from '@radix-ui/react-label'
+import { cn } from '@utils/cn'
+import { cva, type VariantProps } from 'class-variance-authority'
+
+const labelVariants = cva('leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70', {
+  variants: {
+    variant: {
+      default: 'text-sm font-light',
+      sm: 'text-xs font-light'
+    }
+  },
+  defaultVariants: {
+    variant: 'default'
+  }
+})
+
+const Label = React.forwardRef<
```

Comment:
is this component different from the `badge` component that's already added?

Reply:
Yes, it is different. Label is basically an html label vs badge serves a different purpose.

https://ui.shadcn.com/docs/components/label
https://ui.shadcn.com/docs/components/badge